s_tui/StuiBarGraphVector.py

```diff
@@ -186,7 +196,11 @@ def update_displayed_graph_data(self):
 
             graph.set_data(bars, float(max(data_max)))
             y_label_size_max = max(y_label_size_max, graph.get_size()[0])
+        # try:
         s = self.get_label_scale(0, max(data_max), float(y_label_size_max))
+        # except ValueError:
```

Comment:
Just remove the code if it is not needed

Reply:
fixed